SPL/SPL_f.php

```diff
@@ -73,7 +73,7 @@ function spl_autoload_unregister ($autoload_function) {}
 /**
  * Return all registered __autoload() functions
  * @link https://php.net/manual/en/function.spl-autoload-functions.php
- * @return array An array of all registered __autoload functions.
+ * @return array|false An array of all registered __autoload functions.
```

Comment:
Is it `callable[]|false`? I'm not sure

Reply:
- https://www.php.net/manual/en/function.spl-autoload-unregister.php

- https://github.com/php/php-src/blob/master/ext/spl/php_spl.c#L751

... so I think "bool" is correct here?